app/controllers/bike_share_app.rb

```diff
@@ -91,6 +91,10 @@ class BikeShareApp < Sinatra::Base
 
   get "/conditions-dashboard" do
     @conditions = Condition.all
+    @condition_temp_data = Condition.collect_descriptors_for_each_ten_degree_temp_range
+    @condition_precip_data = Condition.collect_descriptors_for_each_precipitation_range
+    @condition_wind_speed_data = Condition.collect_descriptors_for_each_mean_wind_speed_range
+    @condition_visibility_data = Condition.collect_descriptors_for_each_mean_visibility_range
     erb :"/conditions/dashboard"
```

Comment:
this is awesome

Reply:
it makes so much more sense to me now that we're only passing the actual results of the query to the view instead of making the bajillion db queries when a user goes to the page